protostar/commands/test/test_collector.py

```diff
@@ -131,7 +133,17 @@ def _get_test_suite_paths(self, target: Path) -> Generator[Path, None, None]:
     def _collect_test_case_names(
         self, preprocessed: StarknetPreprocessedProgram
     ) -> List[str]:
-        return self._starknet_compiler.get_function_names(preprocessed, prefix="test_")
+        return self._starknet_compiler.get_function_names(
+            preprocessed, predicate=lambda fn_name: fn_name.startswith("test_")
```

Comment:
maybe filter would be better name than predicate?

Reply:
`filter` collides with the global `filter` function